app/models/medium.rb

```diff
@@ -726,7 +720,7 @@ def locked?
     released == "locked"
   end
 
-  def restricted? # rubocop:todo Lint/DuplicateMethods
+  def restricted?
```

Comment:
Can you confirm my observation that with the removal of the first `restricted?` method, the `restricted?` methods for courses and lessons are no longer needed (whereas the `restricted?` method for lectures is still in use).